pkgs/applications/misc/navit/navit.xml

```diff
@@ -0,0 +1,7324 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
```

Comment:
Did you submit this by accident?

Reply:
i'd like to introduce a better config for nixos.

Reply:
Better than what? Does that really need a 7k line xml file? Where does that file come from?

Reply:
We cannot accept that large files as it would affect the channel and repository size.